### DISCUSSION TITLE
Latest php8.3 docker tag?

### DISCUSSION THREAD

**Discussion by twarkie** [php/frankenphp#970] [Q&A] [answered]
I've been using the docker tag `latest-php8.3` for a while but noticed now that it hasn't been updated for 3 months. Do I have to manually move up the version number on each release or is there another tag I should use for the latest php 8.3 version (that I can't find on my own)?

**Comment by dunglas** [comment, marked as answer]
We changed the naming scheme, just use `php8.3` now.
Sorry about that.

**Reply by twarkie** [reply]
No worries. Thank you for clarifying.
